### DISCUSSION TITLE
Flexible columns in app designer

### DISCUSSION THREAD

**Discussion by zzgvh** [Budibase/budibase#1621] [Ideas]
### Issue
I tend to use descriptive names for my components and they can become quite long. This results in their names being fully visible in the left hand column of the designer interface.
### Suggestion
Implement flexible width for the left and possibly also the right hand columns of the interface. Ideally by being able to drag the border between the UI sections.

**Comment by joebudi** [comment]
Thank you @zzgvh a very valid, and expected idea :-). Will raise it to the team today.
